### DISCUSSION TITLE
How to move a group of nodes

### DISCUSSION THREAD

**Discussion by Equiphract** [freeplane/freeplane#920] [Help] [answered]
I am a new FreePlane user and one of the first things I wanted to do is to move all children of a node a little further away from their parent in order to make the map visually more appealing (for me):

![grafik](https://user-images.githubusercontent.com/31915427/210384205-5539630e-06c5-49df-becc-4b100329400c.png)

I did some research and saw some threads asking about this or a similar feature, the answers however were not all that clear to me. Also these threads were close to a decade old so I figured I'd ask again here.

**Comment by dpolivaev** [comment, marked as answer]
The feature is scheduled for Freeplane 1.11.1
